PersonsApi/Setup/HealthCheckSetup.cs

```diff
@@ -0,0 +1,16 @@
+using Dfe.Academies.Infrastructure;
```

Comment:
Lets please add the AddHealthChecks method in the InfrastructureServiceCollectionExtensions in the Infrastructure project. No need for the AddDbHealthChecks
Something like this would do

```
  public static void AddHealthChecks(this IServiceCollection services) {
      services.AddHealthChecks()
            .AddDbContextCheck<MstrContext>("Academies Database");
  }

```